### PR TITLE
build: do not create the bundles when updating the public API

### PR DESCRIPTION
**Pull request by vicb**
- 1.7x faster on my machine (2.7 vs 4.6 min)
- should solve the memory issue

/cc @ocombe

### CODE REVIEW

**Review by jasonaden — APPROVED** [review]
lgtm

**Comment by vicb** [comment]
@IgorMinar could you please approve

File: tools/gulp-tasks/build.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 
-module.exports = (gulp) => (done) => {
+module.exports = {
+  'build': (gulp) => (done) => execBuild(done),
```
Comment (IgorMinar) [inline tools/gulp-tasks/build.js]: can you call this "full" or something like that?

**Review by IgorMinar — APPROVED** [review]
lgtm except for one nitpick

File: tools/gulp-tasks/build.js (hunk 1 above)
Comment (vicb) [inline tools/gulp-tasks/build.js]: "all" ?

**Pull request merged**